css/windowControls.css

```diff
@@ -41,21 +41,21 @@ body.mac .window-drag-area {
 
```

Comment:
`window-drag-area` should probably be hidden on mac when the separate titlebar is enabled.

Reply:
looks like it has no impact but I've just changed it as that probably makes more sense.